index.css

```diff
@@ -0,0 +1,569 @@
+
+}
+
+#divInput input{
+width: 550px;
+height: 25px;
+padding-left: 5px;
+margin-left: 40px;
+border: 1px solid gray;
+
+}
+#divInput button {
+width: 75px;
+height: 28px;
+border-top-right-radius: 2px;
+border-bottom-right-radius: 2px;
+border: 1px solid gray;
+}
+
+#mag{
+  background-image: url('https://archive.vn/Bss88/73e7cc59947d2ad243d63294411f83990629f7fc.png');
+  background-position: -28px -66px;
+  padding-left: 20px;
+  vertical-align: middle;
+  opacity: 0.5;
+}
+#cam {
+  width: 30px;
+  margin-left: 5px;
+  opacity: 0.75;
+}
+#bell {
+  width: 20px;
+  margin-right: 15px;
+  opacity: 0.75;
+}
+
+
+#arrow{
+  background-image: url('https://archive.vn/Bss88/73e7cc59947d2ad243d63294411f83990629f7fc.png');
+  background-position: -125px -314px;
+  padding-left: 20px;
+  vertical-align: middle;
+  height: 25px;
+  margin: 15px 15px 15px 10px;
+  opacity: 0.5;
+
+}
+
+#SignIn {
+  display: inline-block;
+  font-family: roboto, arial, sans-serif;
+  background-color: #167ac6;
+  color: white;
+  font-size: 11px;
+  height: 28px;
+  border-radius: 2px;
+padding: 0px 10px;
+vertical-align: middle;
+width: 75px;
+border: 1px solid #167ac6;
+margin-right: 25px;
+}
+/*LANGUAGE BAR*/
+#languageH p {
+  margin-left: 25px;
+  font-size: 15px;
+
+}
+#languageH{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  background: #167ac6;
+  color: white;
+  width: 100%;
+  /**/
+  margin-left: 30px;
+  margin-right: 30px;
+}
+#langLogo{
+  background-image: url('https://archive.vn/Bss88/73e7cc59947d2ad243d63294411f83990629f7fc.png');
+  background-position: -119px -376px;
+  padding-left: 30px;
+  vertical-align: middle;
+  margin-right: 5px;
+  }
+  #langBut{
+
+    background-image: url('https://archive.vn/Bss88/73e7cc59947d2ad243d63294411f83990629f7fc.png');
+    background-position: -53px -701px;
+  background-color: transparent;
+  border: transparent;
+    height: 20px;
+    width: 20px;
+  align-self: center;
+  margin-right: 10px;
+  }
+
+
+
+  #linksH{
+    display: flex;
+    flex-direction: row;
+    width: 1026px;
+    line-height: 20px;
+    text-decoration: none;
+    border: 3px solid #167ac6;
+    justify-content: space-between;
+    font-size: 20px;
+    background-color: white;
+
+  }
+  #linksH p {
+    margin-left: 20px;
+    font-size: 15px;
+  }
+
+  #change {
+    text-decoration: none;
+    color: #167ac6;
+  }
+
+
+  #learn{
+    text-decoration: none;
+    color: #167ac6;
+    font-size: 11px;
+    margin: 10px;
+  }
+
+/* MainSection */
+
+main{
+  display: flex;
+  flex-direction: row;
+  margin-top:20px;
+  margin-right:0px
+
+}
+
+#leftSection{
+  display: flex;
+  flex-direction: column;
+  margin-left:-5px;
+  width: 60%;
+}
+#video{
+  background: white;
+  margin: 5px;
+}
+#vidTitle{
+  background: white;
+  margin: 5px;
+  padding: 15px;
+}
+#vidTitle h1{
+  font-size: 20px;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+#section2 {
+  /* border: 1px solid black; */
```

Comment:
Delete commented code:
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))